src/runtime_src/core/pcie/driver/linux/xocl/subdev/mb_scheduler.c

```diff
@@ -4238,6 +4238,9 @@ static int convert_execbuf(struct xocl_dev *xdev, struct drm_file *filp,
 	if (scmd->opcode != ERT_START_COPYBO)
 		return 0;
 
+	/* If SSV3 M2M subdev is enabled, execbuf API should not be called for copybo */
+	BUG_ON(M2M_CB(xdev));
```

Comment:
sorry, this bug on has to be changed to a failure return with error msg. jsut realized that it's coming from user space, which should not be allowed to crash driver. my bad.

Reply:
fixed, thanks.